js/src/get-started-page/benefits-card/index.scss

```diff
@@ -0,0 +1,57 @@
+.gla-get-started-benefits-card {
+	// Adjust <CardBody> imported from @wordpress/components.
+	// Repeat selector to make it higher priority.
+	.components-card__body.components-card__body {
+		display: flex;
+		flex-direction: row;
+		padding: 0;
+
+		@media (max-width: $break-small) {
+			flex-direction: column;
+		}
+
+		.components-flex__item {
+			display: flex;
+			flex-direction: column;
+			justify-content: center;
+			padding: $grid-unit-50;
+			max-width: 51.5%;
```

Comment:
❔ The layout looks the same without setting `max-width`. What is `max-width` supposed to handle?

Reply:
Ah, I think it was previously we used a SVG file for the image and from the design the image should be slightly smaller than the text on the right hand side. Thanks for spotting it, updated in 9181eb1.